### ISSUE TITLE
Do we have any analytics installed?

### ISSUE THREAD

**Issue by tyohan** [Shuvayatra/migration-android-new#65]
Please invite DMTP team to the analytics account if you have it. Or setup a new one.

**Comment by tyohan** [comment]
@prashantsh can you update us on this?

**Comment by julianmanandhar** [comment]
@tyohan The analytics currently setup is done using my account. And it contains debug data as well which isn't really helpful. So you could perhaps create new google-analytics and send me the trackingId.
I'll then update the app accordingly.

For now I've added you to the current analytics account.

**Comment by tyohan** [comment]
@julianraj i just add you and @prashantsh to our Firebase project, we're going to use Firebase analytics for this case, because Google already change to Firebase for mobile analytics. Can you install the SDK and if possible also track event where they access the content. Like which content they open. This will useful for understanding user behaviour and also which content they like to see on the app.

**Comment by julianmanandhar** [comment]
@tyohan we are using GCM (Google Cloud Messaging) from the google-project "**nepalmigrantworkers**".  This results in generating two config files with conflicting project ids. 

You could import the same google project into firebase **OR** we could use Firebase Cloud Messaging instead of GCM with some small changes to the implementation in both app and shuvayatra server.

**Comment by tyohan** [comment]
@julianraj yes, i think GCM soon will move to FGM. So please move it and implement the Firebase analytics. Thanks

**Comment by tyohan** [comment]
@julianraj do you track event on content open? We wanna know which content open by user, how often they access content, and what kind content they like to access. Please track this event and publish the APK to Play Store. We're thinking to build a web app after this. But we need to understand their behaviour first. cc @prashantsh

**Comment by julianmanandhar** [comment]
@tyohan We have those implemented using the firebase analytics and will probably publish the new release on the next working day.

**Comment by tyohan** [comment]
@julianraj great, let me know if new version live at Google Play. Thanks

**Comment by julianmanandhar** [comment]
@tyohan 
We have run into a small issue regarding tracking events.
We have enabled the analytics to track events with content information. But in order to view/export these custom information on the firebase analytics console, we need to link the project with google's BigQuery. But the service is paid.

**Comment by tyohan** [comment]
@julianraj for now, we don't need to link it with BigQuery. As long we track the event, in suppose to appear on this page where we can filter it based on audience, or user property. I think it's enough to understand which content type people like to access. Later we can link it with BigQuery if we need more insight.

![image](https://cloud.githubusercontent.com/assets/106703/15816385/c0e8bd42-2bfc-11e6-9c1f-f176cd04e1f3.png)

This is some additional information what event we should track https://support.google.com/firebase/answer/6317498?hl=en&ref_topic=6317484
